src/include/sof/trace/trace.h

```diff
@@ -293,6 +293,21 @@ struct tr_ctx {
 		.level = default_log_level,			\
 	}
 
+/* Throttle interval in microseconds */
```

Comment:
it 'trace throttling' what the kernel calls 'rate_limit' ?

If yes, any reason why we would use a different wording?

Reply:
yes, sorry, forgot to mention the original work by Liam, now in the PR description. He called his method throttling, but yes, I think it's quite similar to the kernel rate-limiting and I'd call it that way too

Reply:
Ah ok, must be british/US words. custom/bespoke, zucchini/courgette. tomayto/tomahto, etc. :-)

I think it's a really good addition when we get lots of errors for the same issue (e.g. buffer w/ zero space, etc).

Reply:
yep, my wording - let align it with the kernel to keep is simple.